packages/cli/src/lingui-compile.ts

```diff
@@ -18,9 +18,18 @@ const noMessages: (catalogs: Object[]) => boolean = R.pipe(
   R.all(R.equals<any>(true))
 )
 
-function command(config: LinguiConfig, options) {
+export type CliCompileOptions = {
+  verbose?: boolean
+  allowEmpty?: boolean,
+  typescript?: boolean,
+  watch?: boolean
+  namespace?: string,
+}
+
+export function command(config: LinguiConfig, options: CliCompileOptions) {
   const catalogs = getCatalogs(config)
 
+  // fixme: this is definitely doesn't work
   if (noMessages(catalogs)) {
     console.error("Nothing to compile, message catalogs are empty!\n")
     console.error(
```

Comment:
This doesn't work completely.  I wrote test, and it fails, I think fixing this logic may introduce a breaking change for some users (because some of theirs catalogs would be rejected from compilation).
 I don't know what to do with this, may be delete completely? What is the value of this validation?

Reply:
Is the issue in this `if (noMessages(catalogs))` condition?

Reply:
Yes, I think previously `noMessages` was supposed to receive message catalogs in "plain format" something like: 

```ts
{
 catalog1: {
    mesageId1: "text"
  },
 catalog2: {
    mesageId1: "text"
  }
}
```

But currently it receives instances of `Catalog` class, which always has some internal fields and this check is always passed